mock/mock.spec

```diff
@@ -190,7 +193,7 @@ pylint-3 py/mockbuild/ py/*.py py/mockbuild/plugins/* || :
 
 %files
 %defattr(0644, root, mock)
-%config(noreplace) %{_sysconfdir}/mock/site-defaults.cfg
```

Comment:
We should still own this one, right? It will be just the small one.

Reply:
Correct, in `mock-core-configs`.

Reply:
Yes, the file should not be owned by `mock`.  It would unnecessarily trigger file conflicts when deploying custom configuration as an RPM package (replacement of `mock-core-configs`).

Reply:
Well, in the past, I put it in mock (and not mock-core-configs) because it contains default mock configuration (albeit commented) and it is the first config file read (after setting up the defaults in utils.py). The intention was that if some other packages want to provide other configs, you can provide it in another file. But not in site-defaults.cfg.

Reply:
The problem is that there are no other files that work like `site-defaults.cfg`.  Either you can duplicate the common settings in all mock profiles, or put it somewhere to `$HOME`, which is far from optimal when you deploy the configuration as an RPM package.  Am I missing anything?

Reply:
> Well, in the past, I put it in mock (and not mock-core-configs) because
> it contains default mock configuration (albeit commented) and it is the
> first config file read (after setting up the defaults in utils.py).

Except for (intentional) exceptions, everybody will have the
/etc/mock/site-defaults.cfg files installed after this PR anyways - and
that file references the previous variant anyways below %doc.

But meh, this is certainly a negative side of this PR because some people
do `rpm -qc mock`, they notice the configuration files - go & edit (they
don't have to know there's mock-core-configs).  OTOH, positive thing is
that `rpm -qd mock` now reveals more ...

That said, would it make any sense to experiment with %ghost
/etc/mock/site-defaults.cfg to keep /etc/mock/site-deaults.cfg listed
as configuration file and heal your argument?

> The intention was that if some other packages want to provide other configs,
> you can provide it in another file

This is what was discussed in this PR;  that mock should put the default
configuration below /usr/share/mock, and override it by /etc/mock (similar
thing that is done by systemd configuration).